encoding/av1-break-even-calculator/info.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "AV1 break-even point calculator",
+  "description": "Calculate the break-even point for AV1 encoding",
+  "long_description": "Calculate the break-even point for AV1 encoding, supplementing the existing encoding.",
+  "executable": {
+    "executable": true,
+    "indexfile": "index.html"
+  },
+  "tags": [
+    "AV1",
+    "break-even",
+    "calculator",
+    "encoding",
+    "return on investment"
```

Comment:
noticed that the search doesn't work on staging
not sure why

Reply:
Probably the search index for the static page will only be updated on a live release?

Reply:
yeah, we can just try that